research/src/pages/prototypes/selectmenu.mdx

```diff
@@ -156,18 +156,18 @@ You can replace the default listbox part in a similar fashion:
 </style>
 <selectmenu class="my-custom-select">
   <div slot="listbox">
-    <popup behavior="listbox">
+    <div popup="popup" behavior="listbox">
       <option>Option 1</option>
       <option>Option 2</option>
       <option>Option 3</option>
       <option>Option 4</option>
       <option>Option 5</option>
-    </popup>
+    </div>
   </div>
 </selectmenu>
 ```
 
-The `<popup>` used here is the element proposed at [Popup (Editor's Draft)](http://localhost:8000/components/popup). The element with `behavior="listbox"` is required to be a `<popup>`. Applying `behavior="listbox"` tells the browser to apply select menu listbox behavior to that element: it will be opened when the `<selectmenu>`'s button is clicked, and the user can select `<option>`s inside it with mouse, arrow keys, and touch.
+The `<div popup=popup>` used here is the element proposed at [Popup (Editor's Draft)](http://localhost:8000/components/popup). The element with `behavior="listbox"` is required to be a `<div popup=popup>`. Applying `behavior="listbox"` tells the browser to apply select menu listbox behavior to that element: it will be opened when the `<selectmenu>`'s button is clicked, and the user can select `<option>`s inside it with mouse, arrow keys, and touch.
```

Comment:
I think the text is ok as-is, but it raises a good question: are we going to require that the element with `behavior=listbox` is **exactly** a `<div popup=popup...` or maybe any element that supports `popup=popup`?

@dandclark WDYT? Should we open an issue for this question?

Reply:
Hmm, if we have a dedicated `<listbox>` element it might make sense to limit it to only `<listbox>`.
Otherwise, I think it should probably be `<div>` or `<span>` only. I can't think of any semantically-meaningful elements that would fit.